src/main/java/de/dennisguse/opentracks/sensors/BluetoothUtils.java

```diff
@@ -119,6 +124,17 @@ public static Integer parseHeartRate(BluetoothGattCharacteristic characteristic)
         return null;
     }
 
+    public static Integer parsePressure(BluetoothGattCharacteristic characteristic) {
```

Comment:
Could we introduce a data class for this?
It is not that obvious that this is 0.1 pa rather than hPA.

Something like 
`AirPressure` that just has the value and provides a get-function like `int getHPA()`?

Reply:
Again separate commit as it makes the review way easier :)